corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -189,46 +187,69 @@ <h4>[Support only] Assigned Locations</h4>
-    {% if request|toggle_enabled:"SUPPORT" %}
-      <div class="tab-pane" id="user-support">
-        <div class="panel-body">
-          <table class="table table-striped table-hover">
-            <tbody>
-              <tr>
-                <td>Created On</td>
-                <td>{{ couch_user.created_on }}</td>
-              </tr>
-              <tr>
-                <td>Last Modified</td>
-                <td>{{ couch_user.last_modified }}</td>
-              </tr>
-              <tr>
-                <td>Last Login</td>
-                <td>{{ couch_user.last_login }}</td>
-              </tr>
-              <tr>
-                <td>Devices</td>
-                <td>
-                  <ul class="list-unstyled">
-                    {% for device in couch_user.devices %}
-                      <li>{{ device.last_used }} - {{ device.device_id }}</li>
+    <div class="tab-pane" id="user-metadata">
+      <div class="panel-body">
+        <table class="table">
+          <tbody>
+            <tr>
+              <td>{% trans "Created On" %}</td>
+              <td>{{ couch_user.created_on|to_user_time:request }}</td>
+            </tr>
+            <tr>
+              <td>{% trans "Last Modified" %}</td>
+              <td>{{ couch_user.last_modified|to_user_time:request }}</td>
+            </tr>
+            <tr>
+              <td>{% trans "Last Login" %}</td>
+              <td>{{ couch_user.last_login|to_user_time:request }}</td>
+            </tr>
+            <tr>
+              <td>
+                {% trans "Last Sync" %}
+                <span class="hq-help-template"
+                  data-content="{% blocktrans %}
+                    Approximate date/time of the last time this user synced with the server from CommCare.
+                  {% endblocktrans %}">
+                </span>
+              </td>
+              <td>{{ couch_user.reporting_metadata.last_sync_for_user.sync_date|to_user_time:request }}</td>
+            </tr>
+            <tr>
+              <td>{% trans "Last Form Submission" %}</td>
+              <td>{{ couch_user.reporting_metadata.last_submission_for_user.submission_date|to_user_time:request }}</td>
+            </tr>
+            <tr>
+              <td>
+                {% trans "Devices" %}
+                <span class="hq-help-template"
+                  data-content="{% blocktrans %}
+                    List of devices used to access CommCare applications, with approximate last-use dates.
+                  {% endblocktrans %}">
+                </span>
+              </td>
+              <td>
+                <table class="table-condensed table-bordered">
+                  <thead>
+                    <tr>
+                      <th>{% trans "Device ID" %}</th>
+                      <th>{% trans "Last Used" %}</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {% for device in couch_user.devices|dictsortreversed:"last_used" %}
```

Comment:
Is this information limited to the request domain or is it showing devices for all domains?
Same for other metadata, is it limited to the active domain only?

Reply:
This is for mobile workers only, so it's all constrained to the domain